### PR TITLE
feat: adding in keychain management

### PR DESCRIPTION
**Pull request by kai0x**

### CODE REVIEW

File: cli.js [hunk 1]
```diff
@@ -176,22 +178,28 @@ yargs(hideBin(process.argv))
...
+            
+            // check for password from user in keychain
+            let password = await keytar.getPassword('armada-cli', walletAddress)
+            if (!password) {
+                let res = await inquirer.prompt([{
+                    name: 'password',
+                    message: 'Enter the password of the wallet.',
+                    type: 'string',
+                    }])
+                password = res.password
```
Comment (sbaranov) [inline cli.js:196]: Missing semi

**Review by sbaranov — APPROVED** [review]

File: cli.js (hunk 1 above)
Comment (kai0x) [reply cli.js:196]: yep will fix all these in the prettier pr

**Comment by kai0x** [comment]
ah I think a rebase messed these up. i'll spin up a new pr instead

**Comment by kai0x** [comment]
added all the same changes to https://github.com/dispersed-labs/dispersed-cli/pull/17. I'll go ahead and close this pr

**Pull request closed without merging**